.github/workflows/publish-docker.yml

```diff
@@ -27,8 +29,12 @@ jobs:
         uses: docker/metadata-action@v3
         with:
           images: getmeili/meilisearch
+          # Output 'latest' tag only when 'release' happen, instead of when 'push tag' event happend (default behavior).
+          # See https://github.com/docker/metadata-action#latest-tag
           flavor: latest=false
```

Comment:
We need to keep the `flavor: latest=false`, according to [Latest tag](https://github.com/docker/metadata-action#latest-tag):

> _latest tag is handled through the [flavor input](https://github.com/docker/metadata-action#flavor-input). It will be generated by default (auto mode) for:_
>
> -   _[type=ref,event=tag](https://github.com/docker/metadata-action#typeref)_
> -   _[type=semver,pattern=...](https://github.com/docker/metadata-action#typesemver)_
> -   _[type=match,pattern=...](https://github.com/docker/metadata-action#typematch)_